### ISSUE TITLE
NAR hash mismatch in input

### ISSUE THREAD

**Issue by hansemschnokeloch** [loophp/nix-shell#212]
Hi,

I have "NAR hash mismatch" errors If I create a new .envrc file in a directory or if I change the php version in an existing .envrc file

```
direnv: loading /home/dev/DGD/.envrc                                                                                                                                        
direnv: using flake github:loophp/nix-shell#env-php74 --impure
error: NAR hash mismatch in input 'https://www.php.net/releases/active?narHash=sha256-20qIiLPpROWf6onLj1r7dMjt%2fVhUO8%2f%2ffzYYt4KlPYQ=' (/nix/store/3vkk556lnayd123wr6rzjvdw6ambsbcs-source), expected 'sha256-20qIiLPpROWf6onLj1r7dMjt/VhUO8//fzYYt4KlPYQ=', got 'sha256-Omnc2u11Y6CnV3oO1NwrzhB+scKXeOx0tkoAZz7PS3g='
direnv: nix-direnv: renewed cache
direnv: export +CONFIG_SHELL +HOST_PATH +IN_NIX_SHELL +NIX_BUILD_CORES +NIX_CFLAGS_COMPILE +NIX_ENFORCE_NO_NATIVE +NIX_LDFLAGS +NIX_STORE +SOURCE_DATE_EPOCH +__structuredAttrs +buildInputs +buildPhase +builder +cmakeFlags +configureFlags +depsBuildBuild +depsBuildBuildPropagated +depsBuildTarget +depsBuildTargetPropagated +depsHostHost +depsHostHostPropagated +depsTargetTarget +depsTargetTargetPropagated +doCheck +doInstallCheck +dontAddDisableDepTrack +mesonFlags +name +nativeBuildInputs +out +outputs +patches +phases +preferLocalBuild +propagatedBuildInputs +propagatedNativeBuildInputs +shell +shellHook +stdenv +strictDeps +system ~PATH ~XDG_DATA_DIRS

```

Any ideas how to solve this ?

Thx

**Comment by hansemschnokeloch** [comment]
Wow that was fast ! Thx a lot

**Comment by hansemschnokeloch** [comment]
Fixed by #213
